### PR TITLE
Refactor TokenConversion class

### PR DESCRIPTION
**Pull request by anighanta**
**Description**:

**Related issue(s)**:
Refactor TokenConversion class to not use AccountStore directly.

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/src/test/java/com/hedera/services/store/models/TokenConversionTest.java [hunk 1]
```diff
@@ -1,157 +1,190 @@
...
+	private static final JKey wipeKey = TxnHandlingScenario.TOKEN_WIPE_KT.asJKeyUnchecked();
+	private static final JKey adminKey = TxnHandlingScenario.TOKEN_ADMIN_KT.asJKeyUnchecked();
+	private static final JKey feeScheduleKey = TxnHandlingScenario.TOKEN_ADMIN_KT.asJKeyUnchecked();
+	private static final long tokenSupply = 777L;
+	private static final long autoRenewPeriod = 1234L;
+	private static final String name = "Testing123";
+	private static final String symbol = "T123";
+	private static final String memo = "memo";
+	private static final boolean freezeDefault = true;
+	private static final boolean kycGrantedDefault = true;
```
Comment (ljianghedera) [inline hedera-node/src/test/java/com/hedera/services/store/models/TokenConversionTest.java:69]: nit: these vars are all constants, maybe use the constant convention to name them.

**Review by ljianghedera — APPROVED** [review]
LGTM overall.

File: hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleToken.java [hunk 2]
```diff
@@ -573,6 +574,11 @@ public void setAccountsFrozenByDefault(boolean accountsFrozenByDefault) {
 		this.accountsFrozenByDefault = accountsFrozenByDefault;
 	}
 
+	public void setAccountsKycGrantedByDefault(boolean accountsKycGrantedByDefault) {
```
Comment (abhishek-hedera) [inline hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleToken.java:577]: nit: the parameters could be made `final` where possible.

**Review by abhishek-hedera — APPROVED** [review]

File: hedera-node/src/test/java/com/hedera/services/store/models/TokenConversionTest.java (hunk 1 above)
Comment (anighanta) [reply hedera-node/src/test/java/com/hedera/services/store/models/TokenConversionTest.java:69]: understood. Will update them in the main PR for branch `ci-enable-2143` - PR #2221

File: hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleToken.java (hunk 2 above)
Comment (anighanta) [reply hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleToken.java:577]: Thanks @abhishek-hedera, Will update them in the main PR for branch `ci-enable-2143` - PR #2221

**Pull request merged**